examples/face_recognition_11/README.md

```diff
@@ -0,0 +1,59 @@
+# Example Integration: Face Recognition (1:1)
+
+This example integration uses the [Labeled Faces in the Wild (LFW)](http://vis-www.cs.umass.edu/lfw/) dataset and Face Recognition (FR) workflow to demonstrate how to test and evaluate end-to-end FR (1:1) model pipelines on Kolena. The evaluation stages are: face detection (bbox), keypoint extraction, and recognition.
+
+## Setup
+
+This project uses [Poetry](https://python-poetry.org/) for packaging and Python dependency management. To get started,
+install project dependencies from [`pyproject.toml`](./pyproject.toml) by running:
+
+```shell
+poetry update && poetry install
+```
+
+## Usage
+
+The data for this example integration lives in the publicly accessible S3 bucket `s3://kolena-public-datasets`.
+
+First, ensure that the `KOLENA_TOKEN` environment variable is populated in your environment. See our
+[initialization documentation](https://docs.kolena.io/installing-kolena/#initialization) for details.
+
+This project defines two scripts that perform the following operations:
+
+1. [`seed_test_suite.py`](face_recognition_11/seed_test_suite.py) creates the following test suites:
+
+    - `labeled-faces-in-the-wild :: gender [FR]`, stratified by `gender` using image metadata
+    - `labeled-faces-in-the-wild :: race [FR]`, stratified by `race` using image metadata
+
+    Run this command to seed the default test suite:
+    ```shell
+    poetry run python3 face_recognition_11/seed_test_suite.py
+    ```
+
+    > **NOTE:**  Face bounding box and keypoint ground truths are inferred from [RetinaFace](https://github.com/serengil/retinaface/) as they are not provided in the LFW dataset. Also, for demo purposes, we have subsampled 10,000 pairs, made up of 9,983 images, from LFW.
+
+2. [`seed_test_run.py`](face_recognition_11/seed_test_run.py) tests multiple face recognition models against the test suite above.
+
+    Run this command to evaluate the default models on the `labeled-faces-in-the-wild :: gender [FR]` and `labeled-faces-in-the-wild :: race [FR]` test suites:
+    ```shell
+    poetry run python3 face_recognition_11/seed_test_run.py
+    ```
+
+Command line arguments are defined within each script to specify what model to use and what test suite to seed/evaluate.
+Run a script using the `--help` flag for more information:
+
+```shell
+$ poetry run python3 face_recognition_11/seed_test_run.py --help
+usage: seed_test_run.py [-h] [--models MODELS [MODELS ...]] [--test_suites TEST_SUITES [TEST_SUITES ...]]
```

Comment:
I believe there is another argument to this script. Please check both scripts and copy the exact results from the `--help` output.

Reply:
I don't see any changes made here.